src/sandstorm/update-tool.capnp

```diff
@@ -0,0 +1,54 @@
+# Sandstorm - Personal Cloud Sandbox
+# Copyright (c) 2015 Sandstorm Development Group, Inc. and contributors
+# All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+@0x96c3fff3f4beb8fe;
+# This file contains schemas relevant to the Sandstorm self-updater.
+
+$import "/capnp/c++.capnp".namespace("sandstorm");
+
+struct PublicSigningKey {
+  key0 @0 :UInt64;
```

Comment:
Is there a good reason this is 4 `UInt64` rather than simply a `Data` or a `List(UInt8)`?  Having it not be a simple bytestring raises questions of order and endianness - is key0 the first or the last 64 bits of the key?

It looks like you're not actually using the fields on the reading side anyway; just relying on Cap'n Proto's layout to pack these together at the beginning of the data field, and to be tightly packed?